testing/test_mark.py

```diff
@@ -1022,3 +1025,41 @@ def test_pytest_param_id_requires_string():
 @pytest.mark.parametrize("s", (None, "hello world"))
 def test_pytest_param_id_allows_none_or_string(s):
     assert pytest.param(id=s)
+
+
+def test_ux_eval_syntax_error(testdir):
```

Comment:
The two tests (and `test_invalid_m_option`) are quite similar -- maybe combine them to one test with `@pytest.mark.parametrize`?

Reply:
sure thing!